nix/t1/conversion/sv-to-verilator-emulator.nix

```diff
@@ -34,7 +34,8 @@ let
       "-y"
       "$DWBB_DIR/sim_ver"
       "-Wno-lint"
-      "+define+PRINTF_FD=t1_common_pkg::log_fd"
+      "+define+PRINTF_FD=do_not_use_printf"
```

Comment:
Just delete it?
```suggestion
```

Reply:
If some code accidentally use `printf` instead of `SimLog.printf`, it will cause subtle bugs. This is a check for it and I believe such a check is useful.

After `PRINTF_FD` is reverted, we may find a new way to check it. Keep it now is not harmful.